package.json

```diff
@@ -23,7 +23,7 @@
     "gulp-gjslint": "^0.1.3",
     "jasmine-spec-reporter": "^0.6.0",
     "jsdoc": "^3.3.0-alpha9",
-    "karma": "^0.12.23",
+    "karma": "^0.13.16",
```

Comment:
Assuming it's because of [this error](https://travis-ci.org/boxuk/angular-symfony-translation/jobs/97866532#L237), in which case I'm happy for this dependnecy version to be bumped.